infra/binder/src/test/java/org/apache/shardingsphere/infra/binder/context/statement/dml/SelectStatementContextTest.java

```diff
@@ -409,6 +412,29 @@ private void assertSetWhere(final SelectStatement selectStatement) {
         assertThat(actual.getWhereSegments(), is(Collections.singletonList(whereSegment)));
     }
     
+    @Test
+    void assertFindColumnProjectionWithoutExpandProjections() {
+        SelectStatementContext selectStatementContext = mock(SelectStatementContext.class, RETURNS_DEEP_STUBS);
+        
```

Comment:
Please remove useless blank line.

Reply:
Ack, I ran `spotless:apply` so I thought the formatting should be good.

Is this something that we could potentially consider move into formatter rules or maybe this is the conventional code style that we used here?

Reply:
> Please remove useless blank line.

Has this suggestion been adjusted? @Zhenye-Na